models/mlagility/requirements.txt

```diff
@@ -0,0 +1,2 @@
+torch==2.0.0
+torchvision==0.15.1
```

Comment:
These requirements are also covered by mlagility. Do we need a separate requirements file here?

Reply:
IIUC, mlagility is still using torch<=1.14.0 according to https://github.com/groq/mlagility/blob/main/models/requirements.txt. ONNX needs PyTorch 2.0 here to convert opset_version 18.

Reply:
MLAgility includes a [setup.py](https://github.com/groq/mlagility/blob/main/setup.py) that installs the base dependencies and a [requirements.txt](https://github.com/groq/mlagility/blob/main/models/requirements.txt) that installs additional packages for the mlagility/models.
The setup.py installs torch>=1.12.1 [here](https://github.com/groq/mlagility/blob/main/setup.py#LL46C9-L46C25) which should install torch2.0.